instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -144,12 +144,12 @@ def get_default_span_details(scope: dict) -> Tuple[str, dict]:
         scope: the asgi scope dictionary
 
     Returns:
-        a tuple of the span, and any attributes to attach to the
-        span.
+        a tuple of the span name, and any attributes to attach to the span.
     """
     method_or_path = scope.get("method") or scope.get("path")
```

Comment:
I believe the logic should be: Use the path if it exists. If not, use "HTTP <method>".

Reply:
I understood, based on the current [wsgi implementation](https://github.com/adamantike/opentelemetry-python-contrib/blob/60222c2282914a484f4c6f41ff1018e09e931f9e/instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py#L177-L179), and the [span name convention](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name), that we would always want the span name to be `HTTP {METHOD|PATH}`.

Do you consider that we should prioritize `{PATH}` over `HTTP {METHOD}`, when the path is available?

Reply:
Yeah `HTTP METHOD` is what we default to when there is no route found. Take a look at [django](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py#L69) and [flask](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py#L76).

Reply:
Makes sense, thanks for the heads up! Comments applied.